components/common/hooks/useGetNotifyAdmin.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
@Afsoon we already have this hook here https://github.com/mattermost/mattermost-webapp/blob/f66c278e04177ab27540669b2a140f2a35e69132/components/notify_admin_cta/notify_admin_cta.tsx#L53

Is this new one different?